Update ESLint and Prettier

friederbluemle opened the pull request:
A few project dependency updates, including a major version update of `eslint-config-standard` - Please check individual commits for details and let me know if there are questions.

Thanks for this package!

Closes #241
Closes #242
Closes #243

kristerkari commented:
Thanks, but I'll just let dependabot handle the updating. It's cleaner that way.  :)

Feel free to open another PR if you want some other things than the versions to be fixed/improved.

friederbluemle commented:
Sure thing! I've removed the minor/patch updates to ESLint and Prettier in `package.json` from this PR. The main thing remaining is the major version update of eslint-config-standard (something that obviously can't be automated, due to the new peer dependency). Since this PR now contains quite a bit less code it should be easier to review. I updated the PR description. Please let me know if I should break it down even more. Thanks!
